court_ball_game/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from court.models import Court
+from player.models import BallGame
+
+
+class CourtBallGame(models.Model):
+    id = models.IntegerField(primary_key=True)
+    court = models.ForeignKey(Court, on_delete=models.CASCADE)
+    ball_game = models.CharField(
+        max_length=100,
+        choices=BallGame.choices
+    )
+
+    class Meta:
+        unique_together = ('id', 'court')
```

Comment:
id is already a primary key (so its promises to be unique) also you do not need to explicitly define it better have Django default

To prevent multiple rows in the table with the same "court" and "ball game" values (but diffrent id) , we want to use "unique_together" attribute. This attribute should involve "court" and "ball game" since we want to ensure that there are no duplicate rows with the same values for these two columns.

Reply:
Changed. 
Please check:)